UI/src/view/MassTransitStopForm.js

```diff
@@ -569,6 +596,11 @@
           'vyöhyketieto', //Information Zone
           'alternative_link_id',
           'liitetty_terminaaliin',
+          'tie',
```

Comment:
The property `vyöhyketieto` should be change to 'vyohyketieto' because it was changed on database